pkg/snmp/snmpparse/config_snmp.go

```diff
@@ -48,7 +48,7 @@ type SNMPConfig struct {
 // set default values used by the agent
 func SetDefault(sc *SNMPConfig) {
 	sc.Port = 161
-	sc.Version = "2"
+	sc.Version = "2c"
```

Comment:
Since the goal here is to mimic the SNMP integration, a better fix might be to default to empty string.
SNMP integration doesn't check for version "2" or "2c"
but relies on present of community_string and check if version == "1".

https://github.com/DataDog/datadog-agent/blob/14736d30acaad238612e27de363a37b837e5a8e3/pkg/collector/corechecks/snmp/internal/session/session.go#L81-L87

Reply:
Fixed 🙂